kubernetes/monitoring/README.md

```diff
@@ -0,0 +1,109 @@
+## Monitoring Hypertrace
+To help in monitoring hypetrace, it exposes a bunch of application and resource level metrics by using 
+certain libraries - micrometer (https://github.com/micrometer-metrics/micrometer) + DropWizard 
+(https://github.com/dropwizard/metrics). It is also configured to report all of the metrics to Prometheus 
+as default reporter. Each pod of the `hypertrace-service` and  `hypertrace-ingester` are annotated 
+with `prometheus.io/scrape: true`, so by deploying Prometheus-operator (https://prometheus-community.github.io/helm-charts/) 
+in your kube environment, you can have access to all the metrics. 
```

Comment:
Could we also add instruction to run the prometheus operator and grafana along with ht in k8s.
Maybe a script can be added to do that

Reply:
I won't care a lot about that, as it's a part of standard tooling at most of the companies. It's just deploying an operator in same cluster. I have linked to prometheus operator github repository now which should give detailed instructions about installation.

Reply:
My point was, should we provide a script to setup monitoring for HT